initializr-generator/src/main/java/io/spring/initializr/generator/buildsystem/maven/MavenPlugin.java

```diff
@@ -99,6 +102,17 @@ public Configuration getConfiguration() {
 				: this.configurationCustomization.build();
 	}
 
+	public boolean shouldLoadExtensions() {
+		return this.loadExtensions;
+	}
+
+	/**
+	 * Enables loading plugin extensions.
+	 */
+	public void loadExtensions() {
```

Comment:
The other methods on `MavenPlugin` that map onto XML tags are generally named after the tag. Along those lines, I wonder if this should be named `extensions` rather than `loadExtensions`.